workitem_test.go

```diff
@@ -90,9 +89,9 @@ func TestCreateWI(t *testing.T) {
 	repo := models.NewWorkItemRepository(ts)
 	controller := WorkitemController{ts: ts, wiRepository: repo}
 	payload := app.CreateWorkitemPayload{
-		Name: "some name",
 		Type: "1",
 		Fields: map[string]interface{}{
+			"system.title":"Test WI",			
 			"system.owner": "tmaeder",
```

Comment:
The `owner` field has been renamed to `assignee` in #185. Remember to change it once this gets merged.